test/parallel/test-fs-rmdir-recursive.js

```diff
@@ -10,17 +10,41 @@ let count = 0;
 
 tmpdir.refresh();
 
-function makeNonEmptyDirectory() {
-  const dirname = `rmdir-recursive-${count}`;
-  fs.mkdirSync(path.join(dirname, 'foo', 'bar', 'baz'), { recursive: true });
+function makeNonEmptyDirectory (depth, files, folders, dirname) {
+  fs.mkdirSync(dirname, { recursive: true });
   fs.writeFileSync(path.join(dirname, 'text.txt'), 'hello', 'utf8');
-  count++;
-  return dirname;
+
+  var o = { flag: 'wx' }
+  if (process.version.match(/^v0\.8/))
```

Comment:
Is this a typo? node v0.8.x is ancient and we shouldn't need to care about it anymore...

Reply:
Haha not a typo. This code comes from rimraf which has been around for a long time. Probably safe to remove it though.

Reply:
@iansu I think it's best to remove it for node core.

Reply:
Thanks for the feedback. I removed the check.